.github/workflows/dev-java-publish-sonatype.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           property: release_version
       - name: Build custom release version
         id: release_ver
-        run: echo ::set-output name=value::"${{ steps.ver.outputs.value }}-${{ steps.branch.outputs.branch_name }}-SNAPSHOT"
+        run: echo ::set-output name=value::"${{ steps.ver.outputs.value }}-${{ steps.branch.outputs.branch_name }}-${{ github.run_id }}-SNAPSHOT"
```

Comment:
Why do you need this ID for snapshots?

Reply:
Easier to understand which version was used at some particular moment while developing

Reply:
But it completely breaks the idea of snapshots. Each time we have a new version and we need to update it

Reply:
`Maven checks for a new SNAPSHOT version in a remote repository for **every build**.`

Okay, got it, thanks for pointing this out